### ISSUE TITLE
[A11Y] Heading level issues on the search page

### ISSUE THREAD

**Issue by foxbunny** [indico/indico#5958]
**Describe the bug**

The search page starts with H2, and missing an H1.

**Additional context**

- https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels.html
